### ISSUE TITLE
Compass Panel

### ISSUE THREAD

**Issue by alasta4204** [EDDiscovery/EDDiscovery#3602]
Good morning 

Thank yolu for your good work on this program. I am using ED Discovery 16.1.2.0 from the Manjaro AUR' Unfortunately the compass isn't working I wonder if you have any ideas?  The window pops out and works as described in the videos but it doesn't seem to be getting any data from Elite

Best regards

Alastair

**Comment by robbyxp1** [comment]
Please upgrade to the latest 18.1 version. That one is very old.  Then we can help you.

**Comment by alasta4204** [comment]
Many thanks. Is there a Linux build for this? I couldn't find a more recent
version with Google.

Best

Alastair

**Comment by alasta4204** [comment]
Good afternoon

 I looked around and couldn't find any more recent Manjaro builds of this.
Unfortunately, I havent the knowledge to build from the source code.

I found that the Windows .EXE files install and run using Wine. There is an
error during the installation process, but it appears to be resolved by
restarting the program. Additionally, there is an issue with the
transparency of the compass window, which I suspect is related to Gnome or
Wayland. I resolved this issue by turning off all transparency. Since I
have two monitors, it's not a big deal to display the compass window on the
other monitor.

Go ahead and close this ticket I am happy with this!

Best


Alastair
